src/app/activitydialog.cpp

```diff
@@ -42,6 +42,8 @@ void ActivityDialog::setActivity(const Activity& activity)
 Activity ActivityDialog::activity() const
 {
     Activity activity;
+    activity.id = 0;
```

Comment:
Instead of init these values here, please init in struct declaration (kimaiapi.h)